vsts/sdl.yaml

```diff
@@ -20,6 +20,7 @@ phases:
     displayName: CredScan
     inputs:
       debugMode: false
+      suppressionsFile: vsts/CredScanSuppressions.json
```

Comment:
This PR brought new samples with it, and one of the samples has a false postive credscan case that this suppressions file accounts for